pkg/guestagent/api/guestservice.proto

```diff
@@ -22,11 +24,20 @@ message Event {
 }
 
 message IPPort {
-  string ip = 1;
-  int32 port = 2;
+  string protocol = 1;
+  string ip = 2;
+  int32 port = 3;
 }
 
 message Inotify {
   string mount_path = 1;
   google.protobuf.Timestamp time = 2;
 }
+
+message TunnelMessage {
+  string id = 1;
+  string protocol = 2;
```

Comment:
Could you explain the allowed values in comment lines?

Reply:
Done.